dataschema/entity.py

```diff
@@ -128,6 +129,17 @@ def GetStructuredTypeName(field_cls: type):
     return None
 
 
+def GetStructuredSubType(field_cls: type, index: int = 0):
+    """For a structured type return the subtype that it contains.
+    Normally for a list index would be 0, for a dict 0 for key, 1 for value."""
+    field_cls = GetOriginalType(field_cls)
+    if not hasattr(field_cls, '__origin__'):
```

Comment:
Will this still work if we upgrade to Python 3.7? I read in [this StackOverflow answer](https://stackoverflow.com/a/52664522) that in Python 3.7 we have to check the class parameters/args.

Reply:
We are in python 3.7 or alter